src/view/pages/Login/index.tsx

```diff
@@ -20,12 +20,15 @@ import { MapStateToProps, MapDispatchToProps, connect } from 'react-redux';
 import { images } from '../../assets/';
 import styles from './style';
 import { State } from '../../../store/types';
+import store from '../../../store';
 import { Route } from '../../navigation/navigator';
 import navActions from '../../../store/modules/nav/actions';
+import loginActions from '../../../store/modules/login/actions';
 import createPage from '../../generators/Page/index';
 
 interface OwnState {
   username: string;
+  password: string;
```

Comment:
I know you want the password to not be in memory but for the sake of this branch I've kept it in a working state